app/controllers/plans_controller.rb

```diff
@@ -329,6 +329,33 @@ def set_test
     end
   end
 
+  def request_feedback
+    @plan = Plan.find(params[:id])
```

Comment:
did you consider the case for when a Plan id does not exist?

Reply:
I don't think this is an issue ... or at least not something to address as part of this PR.

In the instance where the plan id does not match a valid record in the DB, Rails presents the user with a 404 (see ugly screenshot below from dmponline-test).

In the case where the plan id was missing from the URL, the code in the controller would never execute because Rails would not be able to route to the appropriate controller method.

![screen shot 2017-11-08 at 1 46 15 pm](https://user-images.githubusercontent.com/1204467/32576251-452cf9ec-c48b-11e7-80bf-e1b8fb1e6475.png)

Reply:
It is a silent fail, i.e. not finding the record raises ActiveRecord::RecordNotFound exception with rails nicely maps to 404 http status however it is 5xx indeed since an error has been raised in the server. Catching that with begin end is not a big deal but if we want to be DRY, we should really have in place an exception module to globally handle any error.

Reply:
agreed. Something in exception module or application_controller.rb or perhaps even an ActiveRecord Concern (new concept in Rails 4.x ... we don't make use of them yet). Can you create a ticket for this? Something we can look into post MVP